bliss/conf/base_config.yaml

```diff
@@ -184,3 +186,10 @@ surveys:
               dec: -0.9316385797930247
         bands: ${encoder.bands}
         reference_band: 1 # DECaLS r-band, DECaLS-indexed (via header)
+    dc2:
+        _target_: bliss.surveys.lsst.DC2
+        data_dir: ${paths.dc2}/calibrated_exposures/r/
+        cat_path: ${paths.dc2}/merged_catalog_bright_all_
+        band: r
+        batch_size: 1
```

Comment:
Batch size isn't really a property of the dataset; it's more like a property of how we're doing training and predictions.